src/views/wishListShare/wishListShare.jsx

```
@@ -0,0 +1,82 @@
+        </div>
+        <section>
+          <div className="row product-section mx-auto container justify-content-lg-between justify-content-around p-0 g-5">
+            {productList.map((item) => (
+              <SingleProductCard
+                img={item.img}
+                title={item.title}
+                description={item.description}
+                currency={item.currency}
+                targetAmount={item.targetAmount}
+                currentAmount={item.currentAmount}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)